cardano-crypto-tests/cardano-crypto-tests.cabal

```diff
@@ -92,6 +92,38 @@ test-suite test-crypto
 
   ghc-options:          -threaded -rtsopts -with-rtsopts=-N
 
+test-suite test-crypto-vectors
+  import:               base, project-config
+  type:                 exitcode-stdio-1.0
+  hs-source-dirs:       vector-tests
+  main-is:              Main.hs
+  other-modules:
+    Test.EcdsaSecp256k1Tests
+    Test.SchnorrSecp256k1Tests
+    Util.Utils
+    Util.StringConstants
+    TestVector.EcdsaSecp256k1Vectors
+    TestVector.SchnorrSecp256k1Vectors
+    TestVector.Vectors
+  build-depends:        base
+                      , bytestring
+                      , text
+                      , utf8-string
+                      , random-bytestring
```

Comment:
This package is deprecated in favor of either `genByteString` or `uniformByteStringM` functions from `random` package.

Reply:
Was not aware of that. Thank you for giving the function name.

Reply:
It was used for unit tests which are removed now. So i removed random-bytestring in this PR. Thanks again it will be helpful in future.